container-images/ramalama/latest/Containerfile

```diff
@@ -13,6 +13,7 @@ RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.n
 RUN /usr/bin/python3 --version
 RUN pip install "huggingface_hub[cli]==0.24.2"
 RUN pip install "omlmd==0.1.4"
+RUN pip install "tqdm==4.66.5"
```

Comment:
Why are we picking particular versions?  This is going to be very difficult to get into Fedora unless these python tools are package into Fedora.

Reply:
I see that we have [huggingface-hub](https://packages.fedoraproject.org/pkgs/python-huggingface-hub/python3-huggingface-hub/) and [tqdm](https://packages.fedoraproject.org/pkgs/python-tqdm/python3-tqdm/) in fedora packages, however I cannot find omlmd. 

@ericcurtin Please help me in answering why particular versions have been picked, and the further steps to fix this.

Reply:
We have options when it comes to packaging. I'm open minded.

Right now the way things are, we are heavily reliant on the container image. Container images don't have Fedora Rawhide, CentOS Stream to act as a pipeline (at least the way it's done in this repo right now), the container image is automatically rebuilt every night.

So without versions specified, we can easily break overnight because of an API breaking change, etc.

But we should think about what will be ramalama dependencies in the rpm. Maybe it's just python3? Maybe it's just python3 and podman?

What we can't do is make everything a strict dependency in the rpm like the nvidia and amd stuff. Because then we have to pull 100G worth of complex dependencies for every use case.

Reply:
Ok, but we need to have renovate watching these and updating them, or we will end up with ancient versions potentially loaded with CVEs.

Reply:
This is where @p5 may be able to help us out.

Don't let conversations like this block you btw @swarajpande5 we can always change things up in a new PR after this one.

Reply:
Hey
So if we need to version things that aren't natively supported by Renovate, we can introduce something called a "regex manager".  This can be configured to read the version numbers from any arbitrary file given it matches a particular pattern (that we configure), and provide it with some metadata about where to find version information about these dependencies.

I originally tried to PR a universal version of this into the [containers/automation](https://github.com/containers/automation/pull/225) repository, but the consensus between the developers there were this belongs inside individual repos first, and if it is demonstrated to be valuable across multiple places, they may consider adopting it as a central configuration.

I'll create a PR tonight or tomorrow with an implementation of this, and you can decide whether you want to use it or something else.